src/components/DragDropImageUploader.jsx

```diff
@@ -98,18 +100,18 @@ const DragDropImageUploader = () => {
         formData.append(`images[${index}]`, image.file);
       });
 
-      const message = await ArtworkService.uploadArtwork(formData);
-      setResponseMessage(message);
-
-      if (message.includes('successfully')) {
+      const response = await ArtworkService.uploadArtwork(formData);
+      setResponseMessage(response);
+      if (response?.message.includes('successfully')) {
         toast.success('Artwork uploaded successfully!'); // Show success toast
         setImages([]);
         setTitle('');
         setDescription('');
         setCategory('');
-        setTimeout(() => {
-          window.location.reload(); // Reload the page
-        }, 2000); // Delay for toast display
+        // setTimeout(() => {
```

Comment:
// you can use the setTimout, then navigate, 

setTimeout(() => {
  navigate(`/artworks/${response?.data?.images[0]?.artwork_id}`)
 }, 2000); // Delay for toast display